runtime/bcutil/cfreader.c

```diff
@@ -815,6 +819,33 @@ readAttributes(J9CfrClassFile * classfile, J9CfrAttribute *** pAttributes, U_32
 		}
 #endif /* J9VM_OPT_VALHALLA_NESTMATES */
 
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+		case CFR_ATTRIBUTE_ValueTypeObjects: {
+			U_16 numberOfObjects;
+
```

Comment:
no spec yet, but I'm almost certain we'll need to a check to see if the attribute has already been defined